client/post-editor/media-modal/detail/detail-preview-videopress.jsx

```diff
@@ -90,6 +128,34 @@ class EditorMediaModalDetailPreviewVideoPress extends Component {
 		}
 	}
 
+	play() {
+		if ( ! this.player || ! this.player.state ) {
+			return;
+		}
+
+		if ( typeof this.player.state.play === 'function' ) {
+			this.player.state.play();
+		}
+	}
+
+	pause() {
+		if ( ! this.player || ! this.player.state ) {
+			return;
+		}
+
+		if ( typeof this.player.state.pause === 'function' ) {
+			this.player.state.pause();
+		}
+
+		let currentTime;
+
+		if ( typeof this.player.state.videoAt === 'function' ) {
+			currentTime = this.player.state.videoAt();
+		}
```

Comment:
we can also turn this into an immutable binding using a safe-faulting getter. note that all of these deeply nested property accesses are invitations for `TypeError`s

```js
const currentTime = get( this, 'player.state.videoAt', () => undefined )();
```